src/components/molecules/Settings/Account/AccountSection/index.tsx

```diff
@@ -34,22 +36,32 @@ const ProfileSection: React.FC<Props> = ({
 }) => {
   const intl = useIntl();
   const [isOpen, setIsOpen] = useState(false);
-  const [currentLang, setCurrentLang] = useState<string>();
-  const [currentThemeLabel, setCurrentThemeLabel] = useState<string>();
-  type Theme = "DARK" | "LIGHT";
-  const themeItems: { key: Theme; label: string; icon: string }[] = [
-    { key: "DARK", label: intl.formatMessage({ defaultMessage: "Dark theme" }), icon: "moon" },
-    { key: "LIGHT", label: intl.formatMessage({ defaultMessage: "Light theme" }), icon: "sun" },
-  ];
-  useEffect(() => {
-    const lang = items.find(item => item.key === intl.locale);
-    setCurrentLang(lang?.label);
-  }, [intl.locale]);
-
-  useEffect(() => {
-    const label = themeItems.find(themeItem => themeItem.key === appTheme)?.label;
-    setCurrentThemeLabel(label);
-  }, [appTheme]);
+
+  const themeItems: { key: Theme; label: string; icon: string }[] = useMemo(
```

Comment:
I don't have confident, do we need to memorize this as well.
Since it's just a variable and any calculation isn't needed. I assume this might take the memory more that without `memo`.

Reply:
My bad, I was thinking since it relies on internationalization wrapping it in a useMemo was good but now that I think about it we never do that with intl (that package updates itself correctly) :P I'll remove the useMemo.

Reply:
Ah, my bad. I forgot I wrapped this in a useMemo because I was getting an eslint warning:
```
The 'themeItems' array makes the dependencies of useMemo Hook (at line 47) 
change on every render. To fix this, wrap the initialization of 'themeItems' in its 
own useMemo() Hook.eslint
```
For now I'll leave it in the useMemo!

Reply:
OK!